glue/clients/histogram_client.py

```diff
@@ -278,8 +281,7 @@ def component(self):
         return self._component
 
     def set_component(self, component):
-        """
-        Redefine which component gets plotted
+        """ Redefine which component gets plotted
```

Comment:
I would prefer to leave this as it was before, i.e.

```
"""
Description.
...
```

This is what most other package that I know of do. Make sure you include a full stop at the end of the description line.